.github/workflows/typespec-requirement.yaml

```
@@ -0,0 +1,32 @@
+name: TypeSpec Requirement
+
+on: pull_request
+
+jobs:
+  TypeSpec-Requirement:
```

```suggestion
  TypeSpec-Requirement:
    name: TypeSpec Requirement
```

You can control the name of the job, including the space by specifying the name property under the job.

---

If this works, we might not need to update openapi-alps at all!